tasks/configure_jcasc/jenkins_casc.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Create JCasC directory
+  file:
+    path: '{{ jenkins2_home_directory }}/{{ jenkins2_jcasc_directory }}'
+    state: directory
+    owner: '{{ jenkins2_user }}'
+    group: '{{ jenkins2_user }}'
+    mode: 0770
+  become: true
```

Comment:
block section can be used to use only one become instead of 2

Reply:
done